sdk/test-utils/recorder/src/defaultCustomizations.ts

```diff
@@ -1,6 +1,5 @@
 import {
   decodeHexEncodingIfExistsInNockFixture,
```

Comment:
(In your local) Can you re-record all of tables tests and try playing them back?
Just to make sure nothing is broken..

Reply:
Re-generated and pushed the new recordings

Reply:
I meant to test in your local only.
No need to re-record if they are not broken.
Can you revert the changes to recordings?